Automattic/newspack-theme
# fix: update theme descriptions and do not strip from style.css files

### All Submissions:






### Changes proposed in this Pull Request:



#2387 introduced minification for built style.css files, but it didn't make the theme description comment blocks important with a `/*!` prefix for child themes. Without the theme descriptions, [release CI jobs fail](https://app.circleci.com/pipelines/github/Automattic/newspack-theme/3806/workflows/63cbcc4f-c0d9-4879-90d0-c472b8e1c2dd/jobs/7444/steps).

This PR adds the important prefixes to all theme description SCSS, and also updates some outdated description args such as supported WP versions, and removing the AMP tag.

### How to test the changes in this Pull Request:

1. Check out this branch
2. Run `npm run build`
3. In each `newspack-*/style.css` built file, confirm that the theme description block appears at the top of the file. It should look like this:

```
@charset "UTF-8";
/*!
Theme Name: Newspack Joseph
Theme URI: https://github.com/Automattic/newspack-theme
Author: Automattic
Author URI: https://newspack.com
Description: The official theme for Newspack, an all-in-one platform that simplifies publishing and drives audience and revenue right out of the box.
Requires at least: 6.6
Tested up to: 6.7
Version: 2.1.0
License: GNU General Public License v2 or later
License URI: http://www.gnu.org/licenses/gpl-2.0.html
Text Domain: newspack-joseph
Tags: Newspack

This theme, like WordPress, is licensed under the GPL.
Use it to make something cool, have fun, and share what you've learned with others.

Newspack Theme is based on Underscores https://underscores.me/, (C) 2012-2018 Automattic, Inc.
Underscores is distributed under the terms of the GNU GPL v2 or later.

Normalizing styles have been helped along thanks to the fine work of
Nicolas Gallagher and Jonathan Neal https://necolas.github.io/normalize.css/
*/
```

### Other information:
